po/fr.po

```diff
@@ -0,0 +1,15296 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Comprehensive Rust 🦀\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2023-04-11 19:31+0200\n"
+"Last-Translator: Olivier Charrez <olivier.charrez@hotmail.com>\n"
+"Language-Team: French <traduc@traduc.org>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/SUMMARY.md:3
+msgid "Welcome to Comprehensive Rust 🦀"
+msgstr "Bienvenue à Rust Compréhensible 🦀"
```

Comment:
@mgeisler Do you mean by comprehensive, do you mean "exhaustive" ? Or do you see the title as a brand, and should not be translated?

If yes to the first question, I'd make the following suggestion
```suggestion
msgstr "Bienvenue le guide complet de Rust 🦀"
```

ou 


```suggestion
msgstr "Bienvenue dans le guide exaustif de Rust 🦀"
```

Reply:
In that case, I would maybe consider a subtilte because Comprehsive in French is often misundertund as being "something that can be understood"

Reply:
> In that case, I would maybe consider a subtilte because Comprehsive in French is often misundertund as being "something that can be understood"

Yeah, perhaps one could add a translated title in parenthesis afterwards? As translators, you're pretty free to add stuff to make things understandable :smile:

Reply:
I replaced everywhere with `Comprehensive Rust(le guide complet de Rust)` :)